src/api/ParticipantManager.API/Models/Episode.cs

```diff
@@ -4,8 +4,8 @@ public class Episode
 {
   public Guid EpisodeId { get; set; }
   public Guid EnrolmentId { get; set; }
-  public string PathwayVersion { get; set; }
-  public string Status { get; set; }
-  public PathwayTypeEnrolment PathwayTypeEnrolment { get; set; }
-  public ICollection<Encounter> Encounters { get; set; }
+  public string? PathwayVersion { get; set; }
+  public string? Status { get; set; }
+  public PathwayTypeEnrolment? PathwayTypeEnrolment { get; set; }
+  public ICollection<Encounter>? Encounters { get; set; }
```

Comment:
Rather than changing this to nullable, initialise the collection here

public ICollection<Encounter> Encounters { get; init; } = new List<Encounter>();